commit 8a0217ffd432e56231b0d1bccda71449bca5f8f6
Author: Mario Limonciello <Mario_Limonciello@dell.com>
Date:   Thu Aug 20 19:43:51 2009 +0200

    hid2hci: remove superfluous bmAttributes match
    
    For the retrigger of the device on remove we were trying to match bmAttributes
    of self powered which is unnecessary.

diff --git a/extras/hid2hci/70-hid2hci.rules b/extras/hid2hci/70-hid2hci.rules
index b332c16..e133afd 100644
--- a/extras/hid2hci/70-hid2hci.rules
+++ b/extras/hid2hci/70-hid2hci.rules
@@ -19,8 +19,7 @@ ENV{DEVTYPE}!="usb_device", GOTO="hid2hci_end"
 # When a Dell device recovers from S3, the mouse child needs to be repoked
 # Unfortunately the only event seen is the BT device disappearing, so the mouse
 # device needs to be chased down on the USB bus.
-ATTR{bDeviceClass}=="e0", ATTR{bDeviceSubClass}=="01", ATTR{bDeviceProtocol}=="01", \
-  ATTR{idVendor}=="413c", ATTR{bmAttributes}=="e0", \
+ATTR{bDeviceClass}=="e0", ATTR{bDeviceSubClass}=="01", ATTR{bDeviceProtocol}=="01", ATTR{idVendor}=="413c", \
   ENV{REMOVE_CMD}="/sbin/udevadm trigger --action=change --subsystem-match=usb --property-match=HID2HCI_SWITCH=1"
 
 # CSR devices
